### PR TITLE
Serving images using cloudfront - cdn

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

**Comment by Vivek205** [comment]
Don't merge

File: src/utility/constants/GetStarted.js [hunk 1]
```diff
@@ -5,35 +5,10 @@ import ThumbsUpDownIcon from "@material-ui/icons/ThumbsUpDown";
 import SettingsIcon from "@material-ui/icons/Settings";
 import CheckCircleIcon from "@material-ui/icons/CheckCircle";
 
-import CardorListViews from "../../assets/images/GetStarted/BrowsingTheMarketplace/CardorListViews.png";
-import ChangeFilterOptions from "../../assets/images/GetStarted/BrowsingTheMarketplace/ChangeFilterOptions.png";
-import SortOptions from "../../assets/images/GetStarted/BrowsingTheMarketplace/SortOptions.png";
-import RatingandRanking from "../../assets/images/GetStarted/BrowsingTheMarketplace/RatingandRanking.png";
-import Search from "../../assets/images/GetStarted/BrowsingTheMarketplace/Search.png";
+const mediaPath = (type, directory, file, extension) =>
```
Comment (chetankothari) [inline src/utility/constants/GetStarted.js]: Looks like this is being used only once. Can this be inlined?

File: src/assets/thirdPartyServices/snet/text_generation/index.js [hunk 2]
```diff
@@ -12,147 +12,94 @@ import InfoIcon from "@material-ui/icons/Info";
...
+    avatar: avatarPath("EricWeinstein", undefined, undefined, "png"),
+  },
+  { key: "hillaryclinton", value: "Hillary Clinton" },
+  { key: "jimmyfallon", value: "Jimmy Fallon" },
+  { key: "joebiden", value: "Joe Biden" },
+  {
+    key: "joerogan",
+    value: "Joe Rogan",
+    image: imgPath("JoeRogan", undefined, undefined, "png"),
+    avatar: avatarPath("JoeRogan", undefined, undefined, "png"),
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/text_generation/index.js]: You are always passing `undefined` for  orgId and serviceId. Can we remove both these params from `imgPath` and `avatarPath` methods. They seem redundant.

**Review by chetankothari — CHANGES_REQUESTED** [review]
Looks like the path generation helper methods could be cleaned up. I have added comments at specific places. We should be good to go once they are addressed.

File: src/utility/constants/GetStarted.js (hunk 1 above)
Comment (Vivek205) [reply src/utility/constants/GetStarted.js]: It is already inline right.. couldn't get you. could you be  more specific..

Comment (Vivek205) [reply src/utility/constants/GetStarted.js]: https://github.com/singnet/refactored-dapp/pull/325/commits/7eab77b820a105d28a7028cdef394ac6facca7e2

File: src/assets/thirdPartyServices/snet/text_generation/index.js (hunk 2 above)
Comment (Vivek205) [reply src/assets/thirdPartyServices/snet/text_generation/index.js]: https://github.com/singnet/refactored-dapp/pull/325/commits/2b93683b989705e5af8d3005611e573d8a4b76ff

**Review by chetankothari — APPROVED** [review]
LGTM

**Pull request merged**